version/version.go

```diff
@@ -25,3 +38,35 @@ type VersionContext struct {
 func (vc *VersionContext) String() string {
 	return fmt.Sprintf("%s <commit: %s>", vc.Version, vc.Commit)
 }
+
+func (vc *VersionContext) LatestReleasedVersion(cmd *cobra.Command, svc VersionClient) (*github.RepositoryRelease, error) {
```

Comment:
This shouldn't need  `*cobra.Command` to function. If you need a context, accept a context.

With that said, I would say we probably don't want this to log in any way. It likely makes more sense to emit an error here, and have the caller figure out what to do with it.

Reply:
Also, does this need to be a mod of the VersionContext struct for any reason? I think there's certainly value here in hard-coding some of the relevant values for our project ( e.g. the org/project values), but we're not accomplishing that by adding this as a method to the struct.

Make this an library function that accepts the parameters it needs to execute, then call it with the proper parameters provided at your call point(s).

If we make this just a generic library call that accepts the client you need to make to resolve the version, it may also simplify testing of this logic.

Reply:
Agreed on the context here. cobra.Command should not have made it out of cmd package.